src/systems/systems/src/simulation/update_context.rs

```diff
@@ -108,6 +128,22 @@ impl UpdateContext {
         self.longitudinal_acceleration
     }
 
+    pub fn lat_accel(&self) -> Acceleration {
+        self.lateral_acceleration
+    }
+
+    pub fn vert_accel(&self) -> Acceleration {
+        self.vertical_acceleration
+    }
```

Comment:
**suggestion**: As these three types of acceleration are related, consider wraping `longitudinal_acceleration`, `lateral_acceleration`, and `vertical_acceleration` into a struct of its own which is contained within `UpdateContext`. This struct could then also contain functions such as returning a m/s^2 `Vector3`.

```rust
context.acceleration().to_ms2_vector();
```
```

Reply:
I definitely like the `LocalAcceleration` change! Putting things in an array in `LocalAcceleration` isn't really necessary. By just having all three values as fields you could get rid of some complexity, so I'd suggest maybe still doing that.